openwisp_monitoring/db/backends/influxdb/client.py

```diff
@@ -0,0 +1,77 @@
+from datetime import datetime
+
+from influxdb import client
+
+from .. import TIMESERIES_DB
+from .creation import DatabaseCreation
+from .exception import DatabaseException
+
+
+class DatabaseClient(DatabaseCreation, DatabaseException):
+    def get_db(self):
```

Comment:
This will return a new instance everytime we write(), read() or query(). Use the singleton pattern, something like:

```
if not self._db:
    self._db = client.InfluxDBClient(...)
return self._db
```